cmake/onnxruntime_unittests.cmake

```diff
@@ -1023,6 +1023,7 @@ if(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS)
       "${TEST_SRC_DIR}/debug_node_inputs_outputs/debug_node_inputs_outputs_utils_test.cc"
       "${TEST_SRC_DIR}/framework/TestAllocatorManager.cc"
       "${TEST_SRC_DIR}/providers/provider_test_utils.cc"
+      "${TEST_SRC_DIR}/framework/test_utils.cc"
```

Comment:
nit: would be nice to keep things coming from the same folder together (i.e. move this up a line)